main.py

```diff
@@ -112,24 +112,23 @@ def produce_kafka_messages(bucket, job_id, fixture_data, args):
     s3_client = boto3.client("s3")
     for s3_key in fixture_data:
         logger.debug(f"Processing key: {s3_key}")
-        line_no = 0
-        for line in (
+        file_no = 0
+        for file in (
             s3_client.get_object(Bucket=bucket, Key=s3_key)["Body"]
             .read()
-            .splitlines()
         ):
-            line_no += 1
+            file_no += 1
             try:
-                data = json.loads(line)
+                data = json.loads(file)
                 db = data["message"]["db"] if "db" in data["message"] else "missingDb"
                 collection = data["message"]["collection"] if "collection" in data["message"] else "missingCollection"
                 topic_name = f"{args.topic_prefix}{job_id}_{db}.{collection}"
             except json.JSONDecodeError as err:
                 logger.error(
-                    f"line {line_no} of {s3_key} contains invalid JSON data: {err.msg}"
+                    f"line {file_no} of {s3_key} contains invalid JSON data: {err.msg}"
```

Comment:
I'd just have this as `f"{s3_key} contains invalid JSON data: {err.msg}"` - the `line` bit makes no sense now that we don't care about lines.  I also don't think it's worth saying something like "file 1 of x was invalid"; the key name should be enough.

Reply:
done